examples/openvino/export_and_infer_openvino.py

```diff
@@ -0,0 +1,192 @@
+# Copyright (c) Intel Corporation
+#
+# Licensed under the BSD License (the "License"); you may not use this file
+# except in compliance with the License. See the license file found in the
+# LICENSE file in the root directory of this source tree.
+
+import argparse
+import sys
+import time
+
+import timm
+
+import torch
+import torchvision.models as torchvision_models
+
+from executorch.backends.openvino.partitioner import OpenvinoPartitioner
```

Comment:
@cavusmustafa This will eventually import the global openvino init https://github.com/ynimmaga/executorch/blob/openvino_backend/backends/openvino/__init__.py, which will lead to import of the nncf. Thus `nncf.torch.disable_patching` context should be enabled before the run of the main function

Reply:
I believe this issue should be fully resolved once it is addressed in nncf side. So, we are moving forward with using `nncf.torch.disable_patching` for now and we updated the code accordingly.